src/utils/PenaltyReferenceSanitizer.ts

```diff
@@ -0,0 +1,10 @@
+import { sanitizeCompany } from 'app/utils/CompanyNumberSanitizer';
+
+export const sanitizePenalty = (penaltyReference: string): string => {
+
+    penaltyReference = penaltyReference.toUpperCase();
+    const prefix = penaltyReference.substring(0,6);
```

Comment:
isn't the prefix 6 characters, therefore this would be `....substring(0,5);` ?

Reply:
Can you please add the type

Reply:
last number isnt inclusive